js/script.js

```diff
@@ -1,74 +1,88 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] I would not recommend disabling linters, instead, try to improve the code standard.